execution_workflows/DaPars/conf/modules.config

```diff
@@ -0,0 +1,42 @@
+/*
+ * --------------------------------------------------
+ *  Config file for defining DSL2 per module options
+ * --------------------------------------------------
+ *
+ *   Available keys to override module options:
+ *       args           = Additional arguments appended to command in module.
+ *       args2          = Second set of arguments appended to command in module (multi-tool modules).
+ *       publish_dir    = Directory to publish results.
+ *       publish_by_id  = Publish results in separate folders by meta.id value.
+ *       publish_files  = Groovy map where key = "file_ext" and value = "directory" to publish results for that file extension
+ *                        The value of "directory" is appended to the standard "publish_dir" path as defined above.
+ *                        If publish_files == null (unspecified)  - All files are published.
+ *                        If publish_files == false               - No files are published.
+ *       suffix         = File name suffix for output files.
+ *
+ */
+
+params {
+    modules {
+        // 'mode' can either be 'differential' or 'identification' only
+        mode = 'differential'
+        'files' {
+            // relative path from DaPars to gtf genome file
+            genome_file = '../../tests/test_data/gencode_2genes_Chr_prefix.vM18.annotation.gtf'
```

Comment:
Does this path **have** to be relative to the current directory or is an absolute path ok (I haven't tested it)?

Reply:
Good point! it doesnt have to be relative path but it is now because of how this is used in run_dapars.nf. Absolute path might be a lot easier since we can easily get it from running pwd on where the gtf file is located. I'll push a change such that this takes in an absolute path instead. What do you think? @SamBryce-Smith